ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -27,7 +27,7 @@ import cats.syntax.applicative._
 import cats.syntax.functor._
 import doobie.free.connection
 
-sealed abstract class Queries {
+sealed abstract class Queries(val tablePrefix: String) {
```

Comment:
Pretty sure you don't use this.

```suggestion
sealed abstract class Queries(tablePrefix: String) {
```

Reply:
true, thanks for the heads up! :smiley: